tsconfig.json

```diff
@@ -14,6 +14,7 @@
     "src/**/*.ts",
     "test/*.ts",
     "test/**/*.ts",
-    "system-test/*.ts"
+    "system-test/*.ts",
```

Comment:
I'm 90% sure this file is auto-generated, so you'll want to add this to `excludes` in `synth.py`.

Reply:
Done.